state/protocol/inmem/encodable.go

```diff
@@ -36,6 +38,7 @@ type EncodableEpoch struct {
 	DKGPhase3FinalView uint64
 	FinalView          uint64
 	RandomSource       []byte
+	TargetEndTime      time.Time
```

Comment:
prefer to use unit time (`uint64`)
```suggestion
TargetEndTime uint64 // desired real-world end time for the epoch in unix time [seconds]

```